static/usr/lib/systemd/system-generators/rtc-sys-time-init-generator

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+# This generator enables rtc-sys-time-init.service by hooking it into
+# systemd-timesyncd.service with symlink in $1/systemd-timesyncd.service.wants 
+# when this is requested by kernel command line flag 'rtc_sys_time_init'.
+
+DEBUG_ENABLE="false"
+SERVICE="rtc-sys-time-init.service"
+SERVICE_PATH="/usr/lib/systemd/system/$SERVICE"
+KERNEL_CMDLINE_ENABLE_FLAG="rtc_sys_time_init"
+HOOK_TO_UNIT="sysinit.target"
+
+log() {
+    echo "$@" >> "/dev/kmsg"
+}
+
+debug() {
+    if [ "$DEBUG_ENABLE" = "true" ]; then
+        log "$@"
+    fi
+}
+
+read_kernel_command_line() {
+    local cmdline="/proc/cmdline"
+    if [ -n "$SYSTEMD_VIRTUALIZATION" ] && [ ! "$SYSTEMD_VIRTUALIZATION" = "none" ]; then
```

Comment:
I do not think we have to care about virtualization. I have not here of core20 being used with systemd-nspawn.

Reply:
I removed this idea from the script